masq_lib/src/constants.rs

```diff
@@ -12,6 +14,11 @@ pub const CURRENT_LOGFILE_NAME: &str = "MASQNode_rCURRENT.log";
 
 pub const MASQ_PROMPT: &str = "masq> ";
 
+pub const ETH_MAINNET_LABEL: &str = "eth";
+pub const ETH_ROPSTEN_LABEL: &str = "eth_t1";
+pub const ETH_RINKEBY_LABEL: &str = "eth_t2";
```

Comment:
Technically this should probably be `"eth_t4"`... For consistency with `"eth-mainent"` I recommend this be `"eth-t4"` or `"eth-rinkeby"`